docs/source/wrappers.rst

```diff
@@ -0,0 +1,499 @@
+be passed if they are present in ``model``'s signature (i.e. there is an argument
+with the same name in ``model``'s signature):
+
+``meta``
+++++++++
+This is a dictionary containing all of the attributes that
+:py:class:`~scikeras.wrappers.BaseWrapper` creates when it is initialized
+These include ``n_features_in_``, ``y_dtype_``, etc. For a full list,
+see the :ref:`scikeras-api` documentation.
+
+``compile_kwargs``
+++++++++++++++++++++++++
+This is a dictionary of parameters destined for :py:func:`tensorflow.Keras.Model.compile`.
+This dictionary can be used like ``model.compile(**compile_kwargs)``.
+All optimizers, losses and metrics will be compiled if classes with routed parameters
+were passed (see :ref:`param-routing`).
+
+``params``
+++++++++++++++++++++++++
+Raw dictionary of parameters passed to :py:func:`~scikeras.wrappers.BaseWrapper.__init__`.
+This is basically the same as calling :py:func:`~scikeras.wrappers.BaseWrapper.get_params`.
+
+
+Arguments to :py:class:`scikeras.wrappers.BaseWrapper`
+------------------------------------------------------
+
+A complete explanation of all arguments and methods of
+:py:class:`.BaseWrapper` are found in the :ref:`scikeras-api` documentation. Here we
+focus on the main ones.
+
+model
+^^^^^
+
+This is where you pass your Keras :class:``tensorflow.keras.Model``
+building function (``model_build_fn``), or ``Model`` instance.
+Unless you are using a pre-instantiated ``Model``, the arguments
+for your model should be passed to :py:func:`.BaseWrapper.__init__`
+as keyword arguments. These will then be passed to
+``model_build_fn`` so that you can use them to build your ``Model``.
+For example, if your model takes the argument
+``hidden_layer_sizes``, the code would look like this:
+
+.. code:: python
+
+    def model_build_fn(hidden_layer_sizes):
+        model = Model(...)
+        ...
+        return model
+
+    clf = KerasClassifier(
+        model=model_build_fn,
+        hidden_layer_sizes=(100,),
+    )
+
+random_state
+^^^^^^^^^^^^
+
+This behaves similar to the same parameter in ``sklearn`` estimators.
+If set to an integer or a :py:class:`~numpy.random.RandomState` instance,
+it will be used to seed the random number generators used to initialize
+the graph and optimizers. Note that use of this parameter may have
+unforeseen consequences since ``TensorFlow`` only has a *global* random
+state.
+
+optimizer
+^^^^^^^^^
+
+Like :py:class:`~tensorflow.keras.Model`, this can be a string,
+optimizer instance or optimizer class. If you pass a class,
+you will be able to specify it's arguments via parameter routing (see
+:ref:`param-routing`).
+
+batch_size
+^^^^^^^^^^
+
+This argument is passed to :py:func:`~tensorflow.keras.Model.fit`. See the the 
+`Keras Model docs`_ for more details.
+
+validation_split
+^^^^^^^^^^^^^^^^
+
+This argument is passed to :py:func:`~tensorflow.keras.Model.fit`. See the the 
+`Keras Model docs`_ for more details.
```

Comment:
I think you can remove all of these two lines sections that basically say "the argument is passed directly to `fit`, see the API docs for more detail." I'm not sure how much they add; you mentioned "see the docs for full details" above.

Instead, maybe say something like

> SciKeras BaseWrapper also accepts `validation_split`, `batch_size`, `warm_start`, `metrics`, `callbacks`, `verbose` .... See the API docs for more details.

Reply:
I think we should move all of these to the "API Docs" page. I think having them here for now is good as placeholders until we update the docstrings (which will be what generate the API docs). But yeah in the future we should move them from here.

Reply:
I moved these to the current API docs "as is" (I did not edit them to fix references and such).